src/activities/solar_system/Solar_system.qml

```diff
@@ -0,0 +1,340 @@
+                qsTr("After a planet is clicked, the Closeness meter at the bottom-right corner of the screen represents the degree of correctness of your selected answer. The least correct answer is represented by 1%. Try again until you reach a 100% closeness by following the closeness meter, which indicates the correct answer.")
+            ]
+        }
+
+        ListModel {
+            id: containerModel
+        }
+
+        property int itemWidth: horizontalLayout ? background.width / 9 : (background.height - bar.height) / 9
+
+        //Arrangement of all the planets in the solar system
+        GridView{
+            id: planetView
+            y: horizontalLayout ? (parent.height - bar.height) / 2 - cellHeight/2 : 0
+            x: horizontalLayout ? 0 : parent.width / 2 - cellHeight / 2
+            layoutDirection: Qt.LeftToRight
+            verticalLayoutDirection: GridView.BottomToTop
+            width: horizontalLayout ? parent.width : cellWidth
+            height: horizontalLayout ? cellHeight : parent.height - bar.height
+            clip: false
+            interactive: false
+            visible: items.solarSystemVisible
+            cellWidth: background.itemWidth
+            cellHeight: cellWidth
+            model: containerModel
+            delegate: PlanetInSolarModel {
+                planetImageSource: clipImg
+                planetName: bodyName
+            }
+        }
+
+        QuizScreen {
+            id: mainQuizScreen
+            visible: false
+            opacity: items.quizScreenVisible
+            onOpacityChanged: {
+                if(items.quizScreenVisible)
+                    quizScreenAppearAnimation.start()
+                else
+                    quizScreenDisappearAnimation.start()
+            }
+
+            SequentialAnimation {
+                id: quizScreenAppearAnimation
+
+                PropertyAnimation {
+                    target: mainQuizScreen
+                    property: "visible"
+                    to: true
+                    duration: 0
+                }
+
+                NumberAnimation {
+                    target: mainQuizScreen
+                    property: "opacity"
+                    from: 0
+                    to: 1
+                    duration: 200
+                }
+            }
+
+            SequentialAnimation {
+                id: quizScreenDisappearAnimation
+
+                NumberAnimation {
+                    target: mainQuizScreen
+                    property: "opacity"
+                    from: 1
+                    to: 0
+                    duration: 200
+                }
+
+                PropertyAnimation {
+                    target: mainQuizScreen
+                    property: "visible"
+                    to: false
+                    duration: 0
+                }
+            }
+        }
+
+        //Hint dialog while playing the quiz
+        DialogBackground {
+            id: hintDialog
```

Comment:
if you resize the screen, the button overrides the text

Reply:
fixed :+1: